frontend/src/components/EditRecipe/EditRecipeForm.js

```diff
@@ -36,6 +90,36 @@ const EditRecipeForm = ({ history }) => {
             <textarea className="form-control recipe-details" id="description" name="description" value={description} rows="4" onChange={e => setDescription(e.target.value)} />
           </label>
         </div>
+        <div className="search-tag col-6 offset-3">
+          <label className="detail-labels" htmlFor="tags">
+            Tags
+            <Select
+              className="st-search-input"
+              styles={customStyles}
+              value={selectedTags}
+              options={tags.map((tag) => {
+                const obj = {
+                  label: tag.title,
+                  value: tag.title,
+                  id: tag.id,
+                  className: 'hide',
+                };
+                return obj;
+              })}
+              onChange={(selected) => {
+                setSelectedTags(selected);
+              }
+            }
+              placeholder="Search and tag"
+              isMulti
+            />
+            <TagsBar
+              deleteSelTag={handleDelete}
+              tags={selectedTagsWithId}
+              showDeleteButton="yes"
```

Comment:
I think you want `true` here, rather than `"yes"`. To JavaScript, any string is truthy, so even if you put `"no"` here, it'd have the same effect.